docs/src/_reference/settings.md

```diff
@@ -237,10 +237,27 @@ meltano config meltano set project_readonly true
 export MELTANO_PROJECT_READONLY=true
 ```
 
+### <a name="hub-url"></a>`hub_url`
+
+- [Environment variable](/guide/configuration#configuring-settings): `MELTANO_HUB_URL`
+- Default: [`https://hub.meltano.com`](https://hub.meltano.com)
+
+Where Meltano can find the Hub that lists all [discoverable plugins](/concepts/plugins#discoverable-plugins).
+
+This manifest is used by [`meltano discover`](/reference/command-line-interface#discover) and [`meltano add`](/reference/command-line-interface#add), among others.
```

Comment:
> among others

What else is doing the lookup? Don't need to add the caveat unless it's accurate.

Reply:
@tayloramurphy This was copied from the `discovery_url` description, fwiw. But the answer is: Any command that at some point needs the full plugin definition, if there is no lockfile or cached `discovery.yml`. So really any plugin-related command could need to do do the lookup. I listed `meltano discover` and `meltano add` here specifically because they're clearly related to discoverable plugins.

Do you think it's better to omit this entirely, or to expand it to cover what I just explained?

Reply:
```suggestion
This manifest is primarily used by [`meltano discover`](/reference/command-line-interface#discover) and [`meltano add`](/reference/command-line-interface#add). It is also used in cases where the full plugin definition is needed but no lock artifact or cached `discovery.yml` is found.
```

I like adding the above sentence. What do you think @DouweM?

Reply:
@tayloramurphy Works for me!